torch/quantization/fx/quantize.py

```diff
@@ -208,6 +213,13 @@ def _prepare(self, model, qconfig_dict, inplace, is_dynamic_quant):
         # match the patterns that will get quantized
         matches = self._find_matches(model.graph, self.modules, self.patterns)
```

Comment:
perhaps we can move the logic to match custom modules to `_find_matches`, or clearly state in that function that there is also matching logic elsewhere?  Otherwise someone reading the code but not 100% familiar with it can incorrectly assume that `_find_matches` has all the logic to match things.